views/PokemonsView/PokemonsView.jsx

```diff
@@ -35,6 +38,10 @@ function PokemonsView(props) {
     }
   }, []);
```

Comment:
I think we can delete this `if (pageNumber === 0)` condition because that should always be true, right?

Reply:
Not really. For example, this `useEffect` is triggered when navigating back from the Pokemon details page. In that case, as the current `pageNumber` is not equal to `0`, we don't trigger additional `loadPokemons` as they are already "cached" inside the `PokemonsViewModel`.